source/nodejs/adaptivecards-controls/package.json

```diff
@@ -7,6 +7,7 @@
 	"main": "lib/adaptivecards-controls.js",
 	"types": "lib/adaptivecards-controls.d.ts",
 	"style": "dist/adaptivecards-controls.css",
+	"sideEffects": false,
```

Comment:
What does this do?

Reply:
You can find docs [here](https://webpack.js.org/guides/tree-shaking/#clarifying-tree-shaking-and-sideeffects), but the short version is that it tells webpack that simply loading the `adaptivecards-controls` library doesn't cause other side effects (e.g. doing `require("adaptivecards-controls");` in a `<script>` block doesn't create a bunch of DOM elements or override other things that have already been loaded). This allows webpack to better reason about what pieces of JS can be lazy loaded as well as which pieces of JS can be elided altogether.